### ISSUE TITLE
R.A.T. Pro X

### ISSUE THREAD

**Issue by angelus2014** [MayeulC/Saitek#8]
I tried to run the software on my system with the R.A.T. Pro X, but can't even start the python app:

amd ADATA/Saitek-master » sudo ./configurator.py  
Traceback (most recent call last):
  File "./configurator.py", line 12, in <module>
    import usb1

I have installed: 
amd ADATA/Saitek-master » pacman -Qs python3                                1 ↵
local/python 3.5.1-1
    Next generation of the python high-level scripting language
amd ADATA/Saitek-master » sudo pacman -Qs pyqt5
local/pyqt5-common 5.5.1-2
    Common PyQt files shared between python-pyqt5 and python2-pyqt5
local/python-pyqt5 5.5.1-2
    A set of Python 3.x bindings for the Qt5 toolkit
amd ADATA/Saitek-master » sudo pacman -Qs libusb1 
local/lib32-libgusb 0.2.8-1
    GObject wrapper for libusb1
local/libgusb 0.2.8-1
    GObject wrapper for libusb1

So I think I have all the components....

lsusb:
Bus 009 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 013 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 012 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 006 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 011 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 010 Device 003: ID 0738:1718 Mad Catz, Inc. 
Bus 010 Device 002: ID 2516:0011  
Bus 010 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub
Bus 008 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 007 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 005 Device 002: ID 125f:312b A-DATA Technology Co., Ltd. Superior S102 Pro
Bus 005 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 004 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

Angelo

**Comment by MayeulC** [comment]
> amd ADATA/Saitek-master » sudo pacman -Qs libusb1
> local/lib32-libgusb 0.2.8-1
> GObject wrapper for libusb1
> local/libgusb 0.2.8-1
> GObject wrapper for libusb1

I don't think you have libusb1 properly installed. See #7 , I think you will find the answer there. I really need to add this to the readme, though.

Now, for your specific model, I don't see support for your device IDs. I don't own this specific model, so I don't really know if the utility would work with it in its current state. You can give it a go, though, but you will have to modify the code to add your vendor & product ID : 
    0738:1718 
Is the feature set similar to the R.A.T 9, or does it adds in some more functionality?
